client/my-sites/importer/importer-wordpress.jsx

```diff
@@ -39,7 +39,7 @@ export default React.createClass( {
 	render: function() {
 		importerData.description = this.translate(
 			'Import posts, pages, and media ' +
-			'from a WordPress export file.'
+			'from a WordPress export\u00A0file.'
```

Comment:
Is this just to prevent wrapping the last word? It seems like there ought to be a more general solution, but it wouldn't surprise me if not.